packages/contracts-bedrock/src/safe/DeputyPauseModule.sol

```diff
@@ -0,0 +1,213 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
```

Comment:
Can we save a bitof hassled in the future by making this 0.8.25? I think the deps are all floating.

Reply:
SGTM